src/DSharpPlus.VoiceLink/DiscordIpDiscoveryPacket.cs

```diff
@@ -41,6 +41,7 @@ public DiscordIpDiscoveryPacket(ushort type, ushort length, uint ssrc, string ad
             BinaryPrimitives.WriteUInt16BigEndian(dataSpan[2..4], ipDiscovery.Length);
             BinaryPrimitives.WriteUInt32BigEndian(dataSpan[4..8], ipDiscovery.Ssrc);
             Encoding.UTF8.TryGetBytes(ipDiscovery.Address, dataSpan[8..72], out _);
+            dataSpan[71] = 0; // Need to null-terminate the IP string
```

Comment:
Should be null by default

Reply:
True, validation that the string length isn't too long would have been a better solution. Not that submitting an address is ever likely to occur 😅.

Reply:
Yeah a length check at the beginning would've be wiser. Right now the string just gets cut off lol. Still, unlikely to occur and was only implemented for completeness.